nexus/db-queries/src/db/queries/region_allocation.rs

```diff
@@ -566,4 +693,64 @@ mod test {
         db.terminate().await;
         logctx.cleanup_successful();
     }
+
+    #[tokio::test]
```

Comment:
Nice tests!

As a nitpick, neither of these tests needs to be async (they could just be `#[test]`) if you want 'em to be marginally cheaper

Reply:
:+1: be6599b3d